packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform_addition.dart

```diff
@@ -36,5 +52,13 @@ abstract class InAppPurchasePlatformAddition {
   ///   }
   /// }
   /// ```
-  static InAppPurchasePlatformAddition? instance;
+  static InAppPurchasePlatformAddition get instance => _instance;
```

Comment:
Why have you made this non-nullable? Why should a platform with no additions need to set this up?

Reply:
Ah makes sense! Made this nullable.